### ISSUE TITLE
nodeshift invalid options argument

### ISSUE THREAD

**Issue by dudash** [nodeshift/nodeshift#411]
I'm getting a weird error running nodeshift with my webapp. Not sure how to go about debugging.

Heres the command and the output:
```
MacBook-Pro:app-ui dudash$ npx nodeshift --web-app
2020-02-19T22:06:19.223Z INFO loading configuration
2020-02-19T22:06:19.250Z INFO using namespace microservices-demo-dudash at https://api.leonardo.nub3s.io:6443
2020-02-19T22:06:19.252Z WARNING a file property was not found in your package.json, archiving the current directory.
2020-02-19T22:06:19.252Z INFO creating archive of .DS_Store, .gitignore, README.md, app.js, bin, kafka-events.js, keycloak.json, package-lock.json, package.json, public, routes, views
2020-02-19T22:06:19.253Z ERROR invalid options argument
```

Nodeshift version 6.0.0
```
MacBook-Pro:app-ui dudash$ npx nodeshift --version
6.0.0
```

I'm using node 12.16.1
```
MacBook-Pro:app-ui dudash$ node --version
v12.16.1
```
and npm 6.13.4
```
MacBook-Pro:app-ui dudash$ npm --version
6.13.4
```
Just running the resource command I get:
```
MacBook-Pro:app-ui dudash$ npx nodeshift resource
2020-02-19T22:09:36.889Z INFO loading configuration
2020-02-19T22:09:36.920Z INFO using namespace microservices-demo-dudash at https://api.leonardo.nub3s.io:6443
2020-02-19T22:09:36.921Z WARNING No .nodeshift directory
2020-02-19T22:09:36.966Z ERROR error writing files: TypeError: invalid options argument
2020-02-19T22:09:36.969Z ERROR invalid options argument
```

**Comment by lholmquist** [comment]
interesting,  there have been updated versions of `mkdirp` and `tar` added that made it into the nodeshift 6.0.0 release,  can you try running the command with version 5.0.0,   so `npx nodeshift@5.0.0 --web-app`

**Comment by lholmquist** [comment]
I can confirm that 5.0.0 works, but 6.0.0 is broken.   must've been one of those dependencies

**Comment by lholmquist** [comment]
I think i found the error.  We promisifed mkdirp,  but this latest release of that module changed it to return promises, so it messed with what we were doing.   PR'ing a fix

**Comment by lholmquist** [comment]
@dudash  I just released 6.0.1 which should fix the issue
